operator/hack/lokistack_dev.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: loki.grafana.com/v1beta1
+apiVersion: loki.grafana.com/v1
```

Comment:
Should we update the samples in `hack/` to have a schema-config that uses v12 already?